### PR TITLE
feat: add "Key/Value Stores: How We Improved the Performance of Our tx-indexer by 10x"

### PR DESCRIPTION
**Pull request by ajnavarro**
This PR adds the "Key/Value Stores: How We Improved the Performance of Our tx-indexer by 10x" post.

### CODE REVIEW

**Comment by leohhhn** [comment]
Hey, can you please make the folder name after "date_" match the slug (or the other way around)?

File: posts/2024-05-10_kv-stores-indexer/README.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+slug: kv-stores-indexer
+tags: [blog, post, tx-indexer, dev]
+authors: [@ajnavarro]
+---
+
+# Key/Value Stores: How We Improved the Performance of Our tx-indexer by 10x
+
+In this article, we'll discuss how we achieved a tenfold increase in the processing speed of the tx-indexer by applying four key concepts related to our use of key/value storage:
+
+- [Key/Value Stores: How We Improved the Performance of Our tx-indexer by 10x](#keyvalue-stores-how-we-improved-the-performance-of-our-tx-indexer-by-10x)
```
Comment (leohhhn) [inline posts/2024-05-10_kv-stores-indexer/README.md:13]: Unfortunately we havean issue with the rendering of bullet points in `gnoweb`. A workaround for this is to use another character that will be rendered properly: ・

Please replace the bulletpoints here with that character :pray:

**Review by leohhhn — APPROVED** [review]
Left a minor comment, otherwise we're good to go :100:

**Review by zivkovicmilos — APPROVED** [review]
🚀

**Comment by michelleellen** [comment]
Hi @leohhhn and @ajnavarro can we publish this Monday?

**Pull request merged**